src/FirstSteps.Tests/RPG/Adventures/MineTests.cs

```diff
@@ -23,19 +23,23 @@ public void Dwarf_Should_Gain_One_Coin_After_the_First_Visit__In_The_Mine()
         [InlineData(4, 2)]
         [InlineData(5, 3)]
         [InlineData(8, 4)]
-        [InlineData(9, 5)]
-        [InlineData(12, 6)]
-        public void Dwarf_Should_Gain_Three_Coins_When_Numbers_Are_Odd_Otherwise_Should_Gain_One_Coin(int expectedCoins, int visitCounter)
+        [InlineData(11, 5)]
+        [InlineData(14, 6)]
+        [InlineData(15, 7)]
+        [InlineData(18, 8)]
+        [InlineData(19, 9)]
+        [InlineData(22, 10)]
+        [InlineData(23, 11)]
+        [InlineData(26, 12)]
```

Comment:
You did not check 13 visits. Why if it is one of the crucial points of the algorithm?